README.md

```diff
@@ -148,6 +148,7 @@ grant all privileges on database circ to palace;
 #### Environment variables
 
 To let the application know which database to use set the `SIMPLIFIED_PRODUCTION_DATABASE` env variable.
+The `SIMPLIFIED_FCM_CREDENTIALS_FILE` env variable should hold the path to the JSON format GCP service account key
```

Comment:
Can you add a sample fcm credentials file and any other information related to how to set one up so that users don't need to hunt it down?